code/modules/goonchat/browserOutput.dm

```diff
@@ -226,13 +226,13 @@ GLOBAL_DATUM_INIT(iconCache, /savefile, new("data/iconCache.sav")) //Cache of ic
 		C << output(url_encode(url_encode(message)), "browseroutput:output")
 
 /proc/grab_client(target)
-	if(istype(target, /client))
-		return target
-	else if(ismob(target))
+	if(ismob(target))
 		var/mob/M = target
 		if(M.client)
 			return M.client
 	else if(istype(target, /datum/mind))
 		var/datum/mind/M = target
 		if(M.current && M.current.client)
 			return M.current.client
+	else if(istype(target, /client))
```

Comment:
this is moved to the bottom since it is guaranteed to never fire in our compiled codebase (only to_chat uses this proc)